ziti/cmd/ziti/cmd/create_config_router_edge_test.go

```diff
@@ -189,6 +190,51 @@ func TestBlankEdgeRouterNameBecomesHostname(t *testing.T) {
 
 }
 
+func TestDefaultZitiEdgeRouterListenerBindPort(t *testing.T) {
+	expectedDefaultPort := "10080"
```

Comment:
why isn't this a constant from somewhere?

Reply:
though - testing for it to be 10080 hardcoded like this is a good indicator that "doc" probably needs to change. so if that's why- makes sense

Reply:
I believe the main reason I started hard coding values that are actually constants stored elsewhere was because some of these constants are stored in other projects so, if it was changed in another project, this test would fail allowing us to review any changes that might need to be made in this project... it was a discussion from a long time ago but I _think_ that was the main reason.